kairon/shared/pyscript/shared_pyscript_utils.py

```
@@ -120,6 +120,20 @@ def delete_data(collection_id: str, user: Text, bot: Text = None):
             "data": {"_id": collection_id}
         }
 
+    @staticmethod
+    def get_order_details(order_id: str, bot: Text = None) -> dict:
```

Ruff (0.16.2)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)